README.md

```diff
@@ -77,11 +77,8 @@ First checkout the desired tag. Then build the image or sdk by appending the `ka
 # example rt image
 ./kas-container build kas-iot2050-example.yml:kas/opt/preempt-rt.yml:kas/opt/package-lock.yml
 
-# bootloader for advanced board
-./kas-container build kas-iot2050-boot-advanced.yml:kas/opt/package-lock.yml
-
-# bootloader for basic board
-./kas-container build kas-iot2050-boot-basic.yml:kas/opt/package-lock.yml
+# bootloader for the board
+./kas-container build kas-iot2050-boot.yml:kas/opt/package-lock.yml
```

Comment:
Better to keep the downstream compatibility.

Reply:
downstream is only for released uboot and from syfw3.1, there is only one output uboot image for both variant boards. use different yml to build the same image is redundant

Reply:
I think the term `downstream` here is regarding people who is using our meta-iot2050 repo to do their customized applications. So if they started their work based on the old version and once they updates to the new version they will be suprised that their build script will not work anymore due to this change.

I think the current solution is to make symlinks to a common recipe so that both the ymls reproduce same artifact.